openedx/features/learner_profile/static/learner_profile/js/views/learner_profile_view.js

```diff
@@ -118,6 +118,9 @@
                         fieldView.delegateEvents();
                     }
 
+                    // Clear existing content in user profile card
+                    this.$('.profile-section-one-fields').html('');
```

Comment:
It would be good to add a Jasmine test for this case, just so it doesn't regress again.